.github/workflows/test-go.yml

```diff
@@ -318,26 +389,23 @@ jobs:
           # CI commands, like the UI tests, shouldn't have to worry about licensing.
           # So we provide the tests which want an externally supplied license with licenses
           # via the VAULT_LICENSE_CI and VAULT_LICENSE_2 environment variables, and here we unset it.
-          # shellcheck disable=SC2034
-          VAULT_LICENSE=
+          unset VAULT_LICENSE
 
           # Assign test licenses to relevant variables if they aren't already
           if [[ ${{ github.repository }} == 'hashicorp/vault' ]]; then
             export VAULT_LICENSE_CI=${{ secrets.ci_license }}
             export VAULT_LICENSE_2=${{ secrets.ci_license_2 }}
-            export HCP_API_ADDRESS=${{ secrets.HCP_API_ADDRESS }}
-            export HCP_AUTH_URL=${{ secrets.HCP_AUTH_URL }}
-            export HCP_CLIENT_ID=${{ secrets.HCP_CLIENT_ID }}
-            export HCP_CLIENT_SECRET=${{ secrets.HCP_CLIENT_SECRET }}
-            export HCP_RESOURCE_ID=${{ secrets.HCP_RESOURCE_ID }}
-            # Temporarily removing this variable to cause HCP Link tests
```

Comment:
We've removed getting and setting these HCP variables but it seems okay since they were effectively unsed because this variable is unset.